ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -630,8 +559,12 @@ const PanelMain = fp.flow(
             () => (
               <Fragment>
                 <CreatePanel
-                  creatorFreeCreditsRemaining={creatorFreeCreditsRemaining}
-                  profile={profile}
+                  {...{
+                    creatorFreeCreditsRemaining,
+                    profile,
+                    setRuntimeStatus,
+                    status,
+                  }}
```

Comment:
Minor: Personally, I find the `...` notation mixed with non-`...` notation in the same props call confusing. I keep accidentally parsing it as a single object argument instead of several independent props. I think either approach (`...` or no `...`) is fine, but would prefer to not have them combined.

Reply:
and here you can set all the props that way :)

Reply:
Cleaned this up. Thanks for the feedback.